### ISSUE TITLE
Second arg to `assert` is flagged as "not accessed"

### ISSUE THREAD

**Issue by jomido** [microsoft/pyright#270]
**Describe the bug**
If you use a string variable for the second argument in an assert statement, it is not recognized as an "accessed" variable.

**To Reproduce**
```python
def foo():

    msg = "hey, your bar is showing"
    b = 0

    assert b, msg
```

**Expected behavior**
`msg` should not be flagged as "not accessed".

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/1469593/65846252-2ff15000-e30b-11e9-891a-fa3d13dd1aef.png)

**VS Code extension or command-line**
pyright 1.0.67

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.68, which I just published.
